### ISSUE TITLE
signing on ic network does not finish

### ISSUE THREAD

**Issue by malteish** [malteish/ReTransICP#2]
[This call](https://github.com/malteish/ReTransICP/blob/8db3f9d61c529a3a5d4ac30b4330663d3e3b4e78/src/chain_fusion_backend/src/evm_signer.rs#L77-L78) finishes locally, but seems to never return on the ic network.

The log shows all prints up onto this point, but none after, which leads us to this conclusion:
```
$ dfx canister logs chain_fusion_backend  --ic
[42. 2024-06-02T11:40:11.960092814Z]: running job
[43. 2024-06-02T11:40:11.960092814Z]: Successfully started timer for job
[44. 2024-06-02T11:40:42.412757275Z]: Timer has finished, running job now.
[45. 2024-06-02T11:40:42.412757275Z]: Submitting result for job_id: 0
[46. 2024-06-02T11:40:42.412757275Z]: Signing request 
[47. 2024-06-02T11:40:42.412757275Z]: Starting the singing process
[48. 2024-06-02T11:40:42.412757275Z]: Creating the Transaction Request
[49. 2024-06-02T11:40:42.412757275Z]: done with this
[50. 2024-06-02T11:40:42.412757275Z]: Calling sign with ecdsa  EcdsaKeyId { curve: Secp256k1, name: "key_1" }
```

Right now, we do not have any hot leads on how to solve this issue.

**Comment by domwoe** [comment]
The call finishes but there seems to be an issue with the logging functionality. We've reported this.

**Comment by malteish** [comment]
> The call finishes but there seems to be an issue with the logging functionality. We've reported this.

After the call, we receive no further logs, which might be explained by the logging issue you mention. But we also don't have any proof that the canister continues processing the program, as it does not execute any transaction.

@domwoe  can you link to the github issue with the logging please?

**Comment by domwoe** [comment]
Here's the link to the PR with the fix: https://github.com/dfinity/ic/commit/9e7cef791d91470a8479c5631b504bd5b15e982b

Logs within the response handler of the inter-canister call were swallowed.

**Comment by malteish** [comment]
fixed  cd3dc13ffc0eccd6b349776d77f9ca94a6b91fda
